kie-dmn-core/src/test/resources/org/kie/dmn/core/priority_table_context_recursion.dmn

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<semantic:definitions xmlns:semantic="http://www.omg.org/spec/DMN/20151101/dmn.xsd" xmlns="http://www.trisotech.com/definitions/_ff54a44d-b8f5-48fc-b2b7-43db767e8a1c" xmlns:feel="http://www.omg.org/spec/FEEL/20140401" xmlns:triso="http://www.trisotech.com/2015/triso/modeling" xmlns:trisofeed="http://trisotech.com/feed" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" exporter="DMN Modeler" exporterVersion="5.1.8.1" id="_ff54a44d-b8f5-48fc-b2b7-43db767e8a1c" name="not quite all or nothing P" namespace="http://www.trisotech.com/definitions/_ff54a44d-b8f5-48fc-b2b7-43db767e8a1c" triso:logoChoice="Default">
+  <semantic:extensionElements/>
+  <semantic:itemDefinition label="tRiskCategory" name="tRiskCategory">
+    <semantic:typeRef>feel:string</semantic:typeRef>
+    <semantic:allowedValues triso:constraintsType="enumeration">
+      <semantic:text>"High","Low","Medium"</semantic:text>
+    </semantic:allowedValues>
+  </semantic:itemDefinition>
+  <semantic:itemDefinition label="tApprovalStatus" name="tApprovalStatus">
+    <semantic:typeRef>feel:string</semantic:typeRef>
+    <semantic:allowedValues triso:constraintsType="enumeration">
+      <semantic:text>"Approved","Declined","Referred"</semantic:text>
+    </semantic:allowedValues>
+  </semantic:itemDefinition>
+  <semantic:decision id="_3b2953a3-745f-4d2e-b55d-75c8c5ae653c" name="Approval Status" triso:displayName="Approval Status">
+    <semantic:variable id="_7fe30427-4140-4e53-a070-2ecfc723d82b" name="Approval Status" typeRef="tApprovalStatus"/>
+    <semantic:informationRequirement>
+      <semantic:requiredInput href="#_5a4bdb64-f0ef-4978-9e03-6f1ae64a1f17"/>
+    </semantic:informationRequirement>
+    <semantic:informationRequirement>
+      <semantic:requiredInput href="#_8ff18665-84e9-49f2-a8df-8981b1844549"/>
+    </semantic:informationRequirement>
+    <semantic:context>
+        <semantic:contextEntry>
+            <semantic:variable id="m1" name="m1" typeRef="feel:string"/>
+            <semantic:literalExpression>
+                <text>"Hello World"</text>
+            </semantic:literalExpression>
+        </semantic:contextEntry>
+        <semantic:contextEntry>
+            <semantic:context>
+                <semantic:contextEntry>
+                    <semantic:decisionTable hitPolicy="PRIORITY" id="_c98573c0-08b5-4bc3-baf1-3b065fedf1d1" outputLabel="Approval Status">
```

Comment:
The typeRef in line 17 should always be there, although the spec says it is optional. In case it is not, we just set the result type to feel:any and don't do any validation.